### PR TITLE
JFF-1

### PR DESCRIPTION
**Pull request by BtencateSphereon**

### CODE REVIEW

File: packages/jwk-did-provider/__tests__/jwk-did-resolver.test.ts [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+      store: new MemoryDIDStore(),
+    }),
+  ],
+})
+
+describe('@sphereon/jwk-did-resolver', () => {
+  it('should resolve did:jwk', async () => {
+    const didResolutionResult: DIDResolutionResult = await agent.resolveDid({ didUrl: 'did:jwk:eyJrdHkiOiJFQyIsImNydiI6InNlY3AyNTZrMSIsIngiOiJCS2NlRjMwbHBTNkptT1RsS09LQVdudGtKdVRCSzNGX1JoaXlEcTRtdm9jIiwieSI6Im9WY1phQnpiSFJ2UW5iSXhwRWRXbVlRMGtSRm42ajVDRkVQcGxvX09ON1UifQ' })
+
+    expect(didResolutionResult.didResolutionMetadata).toBeDefined()
```
Comment (nklomp) [inline packages/jwk-did-provider/__tests__/jwk-did-resolver.test.ts:40]: Could you check against exact values in this class? 
We then get to catch any regressions.

File: packages/jwk-did-provider/package.json [hunk 2]
```diff
@@ -9,6 +9,7 @@
   },
   "dependencies": {
     "@ethersproject/random": "^5.6.1",
+    "@sphereon/ssi-types": "^0.8.0",
```
Comment (nklomp) [inline packages/jwk-did-provider/package.json:12]: Is there also a reference value in tsConfig?

**Review by nklomp — CHANGES_REQUESTED** [review]
Some small remarks.

I think the tests should match the exact JSON Documents, so we can catch any regressions

File: packages/jwk-did-provider/package.json (hunk 2 above)
Comment (BtencateSphereon) [reply packages/jwk-did-provider/package.json:12]: added

File: packages/jwk-did-provider/__tests__/jwk-did-resolver.test.ts (hunk 1 above)
Comment (BtencateSphereon) [reply packages/jwk-did-provider/__tests__/jwk-did-resolver.test.ts]: changed

**Review by nklomp — APPROVED** [review]
LGTM

**Pull request merged**